twine/commands/upload.py

```diff
@@ -56,6 +57,7 @@ def _make_package(
 ) -> package_file.PackageFile:
     """Create and sign a package, based off of filename, signatures and settings."""
     package = package_file.PackageFile.from_filename(filename, upload_settings.comment)
+    logger = logging.getLogger("LOGGER")
```

Comment:
I think by changing the `setup_logging` to use `twine` instead of `LOGGER`, you could follow the typical convention of using the module name (i.e. `twine.commands.upload`) as the logger name.

```suggestion
    logger = logging.getLogger(__name__)
```

Also, I think this should be at the top of the module, so that any function can use  `logger` without needing to call `getLogger`.

Reply:
Correct @bhrutledge

Reply:
To correctly setup the logger(in `_setup_logging`), we need the name of the logger to be consistent(so that we have the right handlers and `log_level` in place). `__name__` wouldn't work as logger name as `__name__` would be "twine.settings" in `_setup_logging` and `twine.commands.upload` in `upload.py` so the two files would refer to different loggers. Using "twine" presents no issue. Additionally we need to setup the logger` in `settings.py` because   we plan on logging the config file path & config so we cannot wait until the command is called to setup the logger.